common-util/functions/index.jsx

```diff
@@ -100,10 +98,15 @@ export const getAnchor = (title, href, otherProps) => {
  * - elcollectooorr.art
  */
 export const openUrl = url => {
-  window.open(url, (url || '').includes('autonolas.network') ? '_self' : '_blank');
+  window.open(
+    url,
+    (url || '').includes('autonolas.network') ? '_self' : '_blank',
+  );
```

Comment:
Not in scope for this PR, but I would really like to move away from using window.open for links because it creates some very inaccessible experiences.